tests/test_modbus.py

```diff
@@ -9,78 +9,79 @@ class ModbusTestCase(unittest.TestCase):
     # 16bit, all tests are PDU (start from 0 address)
     # INT16,Big Endian
     def test_modbus_datatype_signed_0(self):
-        key = "modbus_read["+self.host+",1,0,3,s]"
```

Comment:
I would keep old test cases as well to ensure your changes are backward compatible with older configurations.

Reply:
There is a new `test_modbus_short_params.py` that will have a portion of old tests with short params.